keyboards/handwired/starrykeebs/dude09/info.json

```
@@ -0,0 +1,40 @@
+{
+    "manufacturer": "StarryKnight",
+    "keyboard_name": "Dude09",
+    "maintainer": "StarryKnight",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": false
+    },
+    "matrix_pins": {
+        "cols": ["C6", "D7", "E6"],
+        "rows": ["D1", "D0", "D4"]
+    },
+    "processor": "atmega32u4",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0002",
+        "vid": "0x534B"
+    },
```

Add Community Layout support
```suggestion
    },
    "community_layouts": ["ortho_3x3"],
```